cerberus/custom_checks/custom_checks.py

```diff
@@ -90,6 +91,26 @@ def check_PV():
         return False
 
 
+def check_kyverno():
+    logging.info("Check if the Kyverno ConfigMap is okay.")
+    result = False
+
+    create_config = runcommand.invoke(
+        'oc create -n kyverno configmap simple-test --from-literal=data="asdf" && echo "successed"')
+    time.sleep(5)
+    update_config = runcommand.invoke(
+        'oc patch configmap/simple-test -n kyverno -p \'{"data":{"data":"dfad"}}\' && echo "successed"')
+    time.sleep(5)
+
+    if ("successed" in create_config) and ("successed" in update_config):
+        delete_config = runcommand.invoke(
+            "oc delete -n kyverno configmap/simple-test")
```

Comment:
We might want to always run the delete, and ignore the result.

If a create works, and a patch fails, currently it wouldnt get deleted. So next run the create will fail as it already exists.

Reply:
But if we have config map in ccm, we don't need create and delete then?

Reply:
Not in CCM, just change the `-n kyverno` to `-n openshift-bcgov-cerberus`

Reply:
oh, got it, I misread your previous msg.